_posts/2018-01-02-project-open-qa.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+img: openqa.png
+title: openQA
```

Comment:
@foursixnine can you double check that the information about openQA is correct? 🙏

Reply:
Yes it is correct